### PR TITLE
Js conf hi 2020 footer

### PR DESCRIPTION
**Pull request by spyhi**
Refactored out footer from index and componentized it, so it can now be reused across pages like header.

Fixed social icons using existing icon library.

### CODE REVIEW

**Comment by spyhi** [comment]
I added a footer to all pages, but it looks wonky on prospectus page and does not behave like live static site. Open to suggestions on fixes.

File: src/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    left: 0;
+    top: 0;
+    width: auto;
+    height: auto;
+    overflow: visible;
+  }
+`;
+
+const Footer = () => {
+  const [state, setState] = useState(false);
```
Comment (jkup) [inline src/components/footer.js]: Are you going to be using these?

File: src/pages/scholarship.js [hunk 2]
```diff
@@ -69,7 +70,7 @@ const Scholarship = () => {
               <h2 className="title">What Are Scholarship Alums Saying?</h2>
               <div className="grid">
                 <div className="grid-half-first">
-                  <blockquote class="twitter-tweet">
+                  <blockquote className="twitter-tweet">
```
Comment (jkup) [inline src/pages/scholarship.js:73]: nice catch!

**Review by jkup — CHANGES_REQUESTED** [review]
Awesome to abstract it! I think just get rid of the useState you aren't using.

File: src/components/footer.js (hunk 1 above)
Comment (spyhi) [reply src/components/footer.js]: Ok, removed the unused imports, skiplink, and useState. No errors this time! 😜

**Comment by spyhi** [comment]
Ok, removed unused imports and code.

One note, I did not import and use layout, so footer component MUST be used inside `Layout` tag wherever it's used. If webpack removes duplicate code (I _think_ that's what it does) then we could bring the Layout tag into the footer and remove the dependency.

**Comment by jkup** [comment]
Looks great! I think the SVG changes somehow made it into this branch? Not sure if those are the changes we already shipped or a different version?

As for Layout, Webpack does not remove duplicate code in this case, it just noop's when it runs into an identical import. So the first time someone imports from 'src/layout.js' it will pull it in and then it will just reference it the following times. Because of this the best practice is to not sweat duplicate imports and call them if and only if the component is needed. So we wouldn't want to import Layout in Footer because it doesn't use it. Does that make sense?

Did you want to remove the SVG changes or is it safe for me to just merge as is?

**Review by jkup — APPROVED** [review]

**Comment by spyhi** [comment]
The SVG changes are safe to merge. I brought them over so I could make sure I didn’t accidentally re-break anything while working on the footer.

Given what you said about Gatsby/Webpack not duplicating imports, we could probably import Layout and wrap the whole footer in a Layout tag to make it more independent, but it’s not something super important to implement just a quality of life thing. You can merge and I can update the layout thing this evening.

**Pull request merged**